pom.xml

```diff
@@ -108,6 +108,19 @@
                     <compilerArgs>--enable-preview</compilerArgs>
```

Comment:
I believe that this line needs to remove to run the build, no?

Reply:
![](https://media.giphy.com/media/kfXJTsTzz0hx6zDfBn/giphy.gif)